packages/react-components/react-carousel-preview/stories/src/Carousel/CarouselAutoplay.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { makeStyles, tokens, typographyStyles } from '@fluentui/react-components';
```

Comment:
I didn't get any audible rotation notification when using NVDA. That doesn't feel right, but maybe regs have changed since the last time I was in Carousel Land.

Reply:
Good callout the audible info is a definitly still a requirement for carousel, this PR mostly just covers the underlying functionality - I'll be adding the overall accessibility props and callouts in this PR: https://github.com/microsoft/fluentui/pull/32181